web/src/components/storage/DeviceSelectionDialog.jsx

```diff
@@ -118,63 +125,70 @@ devices.").split(/[[\]]/);
       inlineSize="large"
       {...props}
     >
-      <Form id="target-form" onSubmit={onSubmit}>
-        <Panels className="stack">
-          <Panels.Options data-variant="buttons">
-            <Panels.Option
-              id={SELECT_DISK_ID}
-              name={OPTIONS_NAME}
-              isSelected={isTargetDisk}
-              onChange={selectTargetDisk}
-              controls={SELECT_DISK_PANEL_ID}
-            >
-              {_("Select a disk")}
-            </Panels.Option>
-            <Panels.Option
-              id={CREATE_LVM_ID}
-              name={OPTIONS_NAME}
-              isSelected={isTargetNewLvmVg}
-              onChange={selectTargetNewLvmVG}
-              controls={CREATE_LVM_PANEL_ID}
-            >
-              {_("Create an LVM Volume Group")}
-            </Panels.Option>
-          </Panels.Options>
-
-          <Panels.Panel id={SELECT_DISK_PANEL_ID} isExpanded={isTargetDisk}>
-            {msgStart1}
-            <b>{msgBold1}</b>
-            {msgEnd1}
-
-            <DeviceSelectorTable
-              aria-label={_("Device selector for target disk")}
-              devices={devices}
-              selected={[targetDevice]}
-              itemChildren={deviceChildren}
-              itemSelectable={isDeviceSelectable}
-              onSelectionChange={selectTargetDevice}
-              variant="compact"
-            />
-          </Panels.Panel>
-
-          <Panels.Panel id={CREATE_LVM_PANEL_ID} isExpanded={isTargetNewLvmVg} className="stack">
-            {msgStart2}
-            <b>{msgBold2}</b>
-            {msgEnd2}
-
-            <DeviceSelectorTable
-              aria-label={_("Device selector for new LVM volume group")}
-              isMultiple
-              devices={devices}
-              selected={targetPVDevices}
-              itemChildren={deviceChildren}
-              itemSelectable={isDeviceSelectable}
-              onSelectionChange={setTargetPVDevices}
-              variant="compact"
-            />
-          </Panels.Panel>
-        </Panels>
-      </Form>
+      <If
+        condition={isLoading}
+        then={ <SectionSkeleton numRows={6} /> }
```

Comment:
Honestly, this skeleton here looks really weird.

As commented during the planning meeting of the card and again today during review, the goal of the skeletons is to "reserve space" in the UI for the future content, so things don't jump up and down as that content is loaded into its place.

In case of a pop-up that is going to load at once as soon the content arrives, that multi-line skeleton looks like some kind of render error. I expected the pop-up to open kind of blank but with some "loading" message and/or a spinner (I personally expected the same three circles we have when loading the whole UI).